test/validate.spec.js

```diff
@@ -61,8 +70,177 @@ describe('Validate', function () {
         },
       ],
     },
+    {
+      certificateName: `invalid_certificate_2`,
+      expectedErrors: [
+        {
+          instancePath: '/Certificate/CommercialTransaction/A97',
+          schemaPath: '#/properties/A97/type',
+          keyword: 'type',
+          params: { type: 'number' },
+          message: 'must be number',
+        },
+        {
+          instancePath: '/Certificate/CommercialTransaction/A98',
+          schemaPath: '#/properties/A98/type',
+          keyword: 'type',
+          params: { type: 'string' },
+          message: 'must be string',
+        },
+        {
+          instancePath: '/Certificate/CommercialTransaction/A99',
+          schemaPath: '#/properties/A99/type',
+          keyword: 'type',
+          params: { type: 'string' },
+          message: 'must be string',
+        },
+        {
+          instancePath: '/Certificate/ProductDescription',
+          schemaPath: '#/additionalProperties',
+          keyword: 'additionalProperties',
+          params: { additionalProperty: 'B14' },
+          message: 'must NOT have additional properties',
+        },
+        {
+          instancePath: '/Certificate/ProductDescription',
+          schemaPath: '#/additionalProperties',
+          keyword: 'additionalProperties',
+          params: { additionalProperty: 'B99' },
+          message: 'must NOT have additional properties',
+        },
+        {
+          instancePath: '/Certificate/Inspection',
+          schemaPath: '#/required',
+          keyword: 'required',
+          params: { missingProperty: 'C00' },
+          message: "must have required property 'C00'",
+        },
+        {
+          instancePath: '/Certificate/Inspection/ChemicalComposition/C71/Actual',
+          schemaPath: '#/definitions/ChemicalElement/properties/Actual/maximum',
+          keyword: 'maximum',
+          params: { comparison: '<=', limit: 100 },
+          message: 'must be <= 100',
+        },
+        {
+          instancePath: '/Certificate/Validation',
+          schemaPath: '#/required',
+          keyword: 'required',
+          params: { missingProperty: 'Z01' },
+          message: "must have required property 'Z01'",
+        },
+        {
+          instancePath: '/Certificate/Validation',
+          schemaPath: '#/required',
+          keyword: 'required',
+          params: { missingProperty: 'Z02' },
+          message: "must have required property 'Z02'",
+        },
+        {
+          instancePath: '/Certificate/Validation',
+          schemaPath: '#/required',
+          keyword: 'required',
+          params: { missingProperty: 'Z04' },
+          message: "must have required property 'Z04'",
+        },
+      ],
+    },
   ];
 
+  const languages = ['DE', 'EN', 'FR', 'PL'];
+  const translationProperties = {
+    certificateFields: [
```

Comment:
This is not a very smart way to test translations, but like mentione [here](https://github.com/s1seven/CoA-schemas/issues/9), at least that's one single source of truth of what properties the translations should contain.